emcc.py

```diff
@@ -1094,6 +1094,12 @@ def get_last_setting_change(setting):
     # Apply -s settings in newargs here (after optimization levels, so they can override them)
     apply_settings(settings_changes)
 
+    if shared.Settings.MINIMAL_RUNTIME or 'MINIMAL_RUNTIME=1' in settings_changes:
+      # Remove the default exported functions 'malloc', 'free', etc. those should only be linked in if used
+      for to_remove in ['malloc', 'free', 'memcpy', 'memset', 'emscripten_get_heap_size']:
```

Comment:
this duplicates the list from `settings.js`. How about instead removing that list from that file, and in emcc.py here check if `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE` is in the changed settings, and handle that accordingly, and differently when minimal runtime?

Reply:
I realize I should be able to just reset `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = []` in here.